File: specification/newrelic/resource-manager/NewRelic.Observability/stable/2022-07-01/examples-js/Monitors_VmHostPayload_MinimumSet_Gen.js
const { NewRelicObservability } = require("@azure/arm-newrelicobservability");
const { DefaultAzureCredential } = require("@azure/identity");

/**
 * This sample demonstrates how to Returns the payload that needs to be passed in the request body for installing NewRelic agent on a VM.
 *
 * @summary Returns the payload that needs to be passed in the request body for installing NewRelic agent on a VM.
 * x-ms-original-file: specification/newrelic/resource-manager/NewRelic.Observability/stable/2022-07-01/examples/Monitors_VmHostPayload_MinimumSet_Gen.json
 */
async function monitorsVMHostPayloadMinimumSetGen() {
  const subscriptionId = process.env["NEWRELICOBSERVABILITY_SUBSCRIPTION_ID"] || "ddqonpqwjr";
  const resourceGroupName = process.env["NEWRELICOBSERVABILITY_RESOURCE_GROUP"] || "rgopenapi";
  const monitorName = "ipxmlcbonyxtolzejcjshkmlron";
  const credential = new DefaultAzureCredential();
  const client = new NewRelicObservability(credential, subscriptionId);
  const result = await client.monitors.vmHostPayload(resourceGroupName, monitorName);
  console.log(result);
}

File: specification/newrelic/resource-manager/NewRelic.Observability/stable/2024-01-01/examples-java/Monitors_VmHostPayload_MinimumSet_Gen.java

/**
 * Samples for Monitors VmHostPayload.
 */
public final class Main {
    /*
     * x-ms-original-file: specification/newrelic/resource-manager/NewRelic.Observability/stable/2024-01-01/examples/
     * Monitors_VmHostPayload_MinimumSet_Gen.json
     */
    /**
     * Sample code: Monitors_VmHostPayload_MinimumSet_Gen.
     * 
     * @param manager Entry point to NewRelicObservabilityManager.
     */
    public static void monitorsVmHostPayloadMinimumSetGen(
        com.azure.resourcemanager.newrelicobservability.NewRelicObservabilityManager manager) {
        manager.monitors().vmHostPayloadWithResponse("rgopenapi", "ipxmlcbonyxtolzejcjshkmlron",
            com.azure.core.util.Context.NONE);
    }
}
